### ISSUE TITLE
[FEATURE] can we add project section?

### ISSUE THREAD

**Issue by smithop** [devvsakib/hackerboi#4]
**Describe**

i was thinking to add a PROJECT page to showcase all contributor projects, for feature.

**Comment by devvsakib** [comment]
> **Describe**
> 
> i was thinking to add a PROJECT page to showcase all contributor projects, for feature.

yesss, we can. do you have any idea?if yes, please start working. good luck

**Comment by inbuilt-aura** [comment]
Hey,can you explain me which kind of project are you asking for?

**Comment by devvsakib** [comment]
> Hey,can you explain me which kind of project are you asking for?

any kind of project built by using react. if you have project you can add them in "PROJECTS" folder

**Comment by inbuilt-aura** [comment]
@devvsakib yeah,i have.I just made a color website,if you want i can add it for you.Plzz count it as a hacktonerfest

**Comment by devvsakib** [comment]
> @devvsakib yeah,i have.I just made a color website,if you want i can add it for you.Plzz count it as a hacktonerfest

sure

**Comment by inbuilt-aura** [comment]
@devvsakib plzz check, i recently pulled my pr request. Plzz merge it by verifying and add me into hacktoberfest-2022.

**Comment by devvsakib** [comment]
> @devvsakib plzz check, i recently pulled my pr request. Plzz merge it by verifying and add me into hacktoberfest-2022.

 sure
